src/priorityq.py

```diff
@@ -9,18 +9,32 @@ def __init__(self):
         """Priority Queue constructor method."""
         self.dict = {}
 
-    def insert(self, key, val):
+    def insert(self, value, priority):
         """Priority Queue insert method."""
-        self.dict.setdefault(key, []).append(val)
+        if not isinstance(priority, int):
+            raise TypeError("Priority must be an integer")
+        if priority in self.dict:
+            self.dict[priority].append(value)
+        else:
+            self.dict[priority] = [value]
+        print(self.dict)
 
     def pop(self):
         """Priority Queue pop method."""
-        mostest_importantestest = self.dict[min(self.dict.keys())][0]
-        if min(self.dict.keys()) < 0:
-            self.dict[min(self.dict.keys())].pop
-        return mostest_importantestest
+        if len(self.dict):
+            cursor = 0
+            while cursor not in self.dict:
+                cursor += 1
```

Comment:
What happens if I insert a priority 1 value and a priority 2 value, then pop, and then pop again?  You don't have a test that covers this, and I don't think the right thing would happen.  Add a test and tell me if it works.